.circleci/config.yml

```diff
@@ -261,3 +291,17 @@ workflows:
             - check_linting
             - check_formatting
             - check-release-version
+
+      - deploy-pre-release:
+          filters:
+            branches:
+              only:
+                - python-3.9
```

Comment:
Happy to leave this, people can change it at will in the future, should they wish to deploy a pre-release for their branch.